docs/Project.toml

```diff
@@ -1,9 +1,14 @@
 [deps]
+ComponentArrays = "b0b7db55-cfe3-40fc-9ded-d10e2dbeff66"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 FHist = "68837c9b-b678-4cd5-9925-8a54edc8f695"
 Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
 Minuit2 = "37821647-3276-4ed8-9a4e-bc9886b3c106"
+Optimization = "7f7a1694-90dd-40f0-9382-eb1efda571ba"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Polynomials = "f27b6e38-b328-58d1-80ce-0feddd5e7a45"
 QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
+
+[compat]
+Plots = "=1.40.14"
```

Comment:
I think Plots.jl have a bug in latest release, see https://github.com/JuliaPlots/Plots.jl/issues/5157